address/address_test.go

```diff
@@ -1,159 +1,104 @@
 package address
 
 import (
-	"encoding/json"
-	"fmt"
+	"math/rand"
 	"testing"
+	"time"
 
-	"gx/ipfs/QmPVkJMTeRC6iBByPWdrRkD3BE5UXsj5HPzb4kPqL186mS/testify/assert"
-)
+	"gx/ipfs/QmPVkJMTeRC6iBByPWdrRkD3BE5UXsj5HPzb4kPqL186mS/testify/require"
+	logging "gx/ipfs/QmbkT7eMTyXfpeyB3ZMxxcxg7XH8t6uXp49jqzz4HB7BGF/go-log"
 
-var hashes = make([][]byte, 5)
+	"github.com/filecoin-project/go-filecoin/bls-signatures"
+	"github.com/filecoin-project/go-filecoin/crypto"
+)
 
 func init() {
-	for i := range hashes {
-		hashes[i] = Hash([]byte(fmt.Sprintf("foo-%d", i)))
-	}
+	logging.SetDebugLogging()
+	rand.Seed(time.Now().Unix())
 }
 
-func TestEmptyAddress(t *testing.T) {
-	assert := assert.New(t)
-	assert.True((Address{}).Empty())
-	assert.False(NewMainnet(hashes[0]).Empty())
-}
+func TestIDAddress(t *testing.T) {
```

Comment:
Let's get some test vectors in here. I want to see a few for ID addresses, like what is the address for ID 0, 1, 10, 150, 999999, and so on. I would expect them to be `f00`, `f01`, `f010`, etc, but let's make it very clear with tests.